cluster-peering-termgw/dc2/resources/web.json

```diff
@@ -0,0 +1,14 @@
+{
+	"Node": "web",
+	"Address": "34.173.103.11",
```

Comment:
Need to make these templates and point out that the address needs to be updated once the service is started